app/src/main/java/com/concordium/wallet/ui/cis2/SendTokenReceiptActivity.kt

```diff
@@ -88,9 +94,9 @@ class SendTokenReceiptActivity : BaseActivity() {
     }
 
     private fun onFinish() {
-        finish()
-        val intent = Intent(this, MainActivity::class.java)
-        intent.addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP)
+        val intent = Intent(this, AccountDetailsActivity::class.java).apply {
+            addFlags(Intent.FLAG_ACTIVITY_CLEAR_TOP)
+        }
         startActivity(intent)
```

Comment:
For my understanding, did the `finish()` end this activity, causing the `startActivity(intent)` to not have any effect? :thinking: